contracts/FTribeVoting.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.8.0;
+
+interface CErc20Delegator {
+    function balanceOf(address owner) external view virtual returns (uint);
+    function exchangeRateStored() external view returns (uint);
+}
+
+contract FTribeVoting {
+
+  CErc20Delegator public FTRIBE = CErc20Delegator(address(0xFd3300A9a74b3250F1b2AbC12B47611171910b07));
+
+  function balanceOf(address who) public view returns (uint256) {
+    return FTRIBE.balanceOf(who) * FTRIBE.exchangeRateStored() / 1e18;
```

Comment:
I'm not sure how much of a problem this is - but could someone could possibly exploit inaccuracies of small-number math here?

ie if FTribe user balance is small (for example a few wei) and the exchange rate is very close to 1e18, we'll get weird results.

Reply:
Exchange rate stored is about 1.02e18 and monotonically increasing as borrowers pay interest.

As long as someone can't artificially massively inflate their balance, this is fine. It is used to calculate the snapshot voting weight.